ElvUI/Core/Core.lua

```diff
@@ -1203,6 +1203,7 @@ function E:Initialize()
 		local msg = format(L["LOGIN_MSG"], self.media.hexvaluecolor, self.media.hexvaluecolor, self.version)
 		if Chat.Initialized then msg = select(2, Chat:FindURL("CHAT_MSG_DUMMY", msg)) end
 		print(msg)
+		print(format("Your role is %s. Use /role [melee, caster, ranged, tank] to change it.", E.Role))
```

Comment:
Isn't there like a E:Print() for this?  The message should indicate it'd from elvui in some way.

Reply:
Also unsure if already the case but if the command doesn't it should assign lfg role as well.

Reply:
It's not related to lfg role normally and only used internally in ElvUI. That is, `caster` could be healer or dps but elvui would still use it to display spell crit.  We could add a healer role though and then use it to also assign lfg roles.

Reply:
But then, lfg allows multiple roles, so i don't think it's the best coupling.

Reply:
I figured caster would just default to damager but it makes sense it wouldn't since it's just for datatexts and stuff.

Maybe some way to differ the wording from role? I imagine it would just be confusing being asked to set a role.

Maybe like 'use /elvrole role to optimize the UI for x role. Your current role is y'